spec.html

```diff
@@ -39417,6 +39152,31 @@ <h1>GeneratorYield ( _iterNextObj_ )</h1>
           1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-yield" aoid="Yield">
+        <h1>Yield ( _value_ )</h1>
+        <p>The abstract operation Yield takes argument _value_. It performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _generatorKind_ be ! GetGeneratorKind().
+          1. If _generatorKind_ is ~async~, then return ? AsyncGeneratorYield(_value_).
+          1. Otherwise, return ? GeneratorYield(! CreateIterResultObject(_value_, *false*)).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-createiteratorfromclosure" aoid="CreateIteratorFromClosure">
+        <h1>CreateIteratorFromClosure ( _closure_, _generatorBrand_, _generatorPrototype_ [ , _generatorSlots_ ] )</h1>
```

Comment:
In the current [Number.range spec](https://tc39.es/proposal-Number.range/#sec-algor) step 19

> Let iterator be ! CreateIteratorFromClosure(closure, ~Range Iterator~, %RangeIteratorPrototype%, « start, end, step, inclusiveEnd »).

I need to store some information on the generator. Because the [getter %RangePrototype.start%](https://tc39.es/proposal-Number.range/#sec-properties-of-the-rangeiterator-prototype-object-start) need that info and it can't read it from an Abstract Closure.

Reply:
It seems like the slots argument of CreateIteratorFromClosure could be added in the Number.range proposal, then?

Reply:
Yeah, agreed, that should be removed from this PR.

Reply:
removed